### ISSUE TITLE
{Adaptive Cards - Expense Report}: Screen Reader is not conveying the Required information when navigating to the Please specify an appropriate reason for rejection * edit field..

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9196]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Win11
OS Version: 24H2 (OS Build 22631.5189)
URL: https://adaptivecards.io/
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on edge dev browser.
2. Home page will appear.
3. Press tab key and navigate to the Sample tab item and select it.
4. Press tab key and navigate to the 'Expense Report' option and press enter key to activate it.
5. Press tab key and navigate to the 'Reject' button and press enter key to activate it.
6. Navigate to edit field and observe the issue.

**Actual Result:**
Screen Reader is not announcing the required information while navigating on the Please specify an appropriate reason for rejection edit field here it is announcing as 'Please specify an appropriate reason for rejection edit field.'

**Expected Result:**
While navigating on the 'Please specify an appropriate reason for rejection ' edit field screen reader should convey the required information as well and it should announce as **'Please specify an appropriate reason for rejection Edit field required.'**

**Note:**
Same issue is repro with NVDA.

**User Impact:**
It will impact screen reader user as they will not be able to know whether the edit field is required or not?

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Screenshots

![Image](https://github.com/user-attachments/assets/7544291e-23fc-4265-a990-e2fbea6766a6)

https://github.com/user-attachments/assets/66b535bc-f6ac-495f-9e05-0c03426a9a3d

### Card JSON

```JSON
NA
```
